src/ansys/dpf/core/core.py

```diff
@@ -412,7 +430,13 @@ def load_library(self, file_path, name="", symbol="LoadOperators", generate_oper
         >>> # base.load_library('meshOperatorsCore.dll', 'mesh_operators')
 
         """
-        file_path = str(file_path)
+        if not name:
```

Comment:
```suggestion
        if name is None:
```
@rafacanton the None is never passed to the server, so there is no expected error.
The server API only accepts a string, which is why we used to pass an empty string.